web/pandas/static/css/pandas.css

```diff
@@ -45,6 +45,12 @@ a.navbar-brand img {
 div.card {
     margin: 0 0 .2em .2em !important;
 }
+@media (min-width: 576px) {
```

Comment:
Out of curiosity. What is this `min-width`?

Reply:
This min-width after the "@media" means that the enclosed rules will only apply if the screen width is _at least_ 576 px. 

i.e., if the screen is less than 576 px, then there are no rules applied. See:
![less_than_576](https://user-images.githubusercontent.com/18680207/118185978-2a2a7e80-b40b-11eb-95fa-67cc1d6efd2b.png)

when the screen is more than 576 px wide, then the rules get applied. See:

![more_than_576](https://user-images.githubusercontent.com/18680207/118186010-357daa00-b40b-11eb-9e2f-bbf23c21d5e4.png)

I can also make it so that the rules always apply no matter the width. See:

![always_fixed_width](https://user-images.githubusercontent.com/18680207/118186324-9c02c800-b40b-11eb-8378-8dc210b6205f.png)

I'm not sure which is better so I just followed the behavior of the existing website.